.travis.yml

```diff
@@ -6,6 +6,7 @@ go:
 install:
   - make dependencies
   - test -z "$(go fmt ./...)"
+  - test -z "$(golint -min_confidence=1.0 ./modules/host)"
```

Comment:
yeah. I'd rather use the Makefile, and the `pkgs` variable, but then you have to have golint passing for every package. Otherwise, it's going to check every package every time you run `make test-long`, which could cause slowdown especially if you are using `run=` to minimize the amount of time.

It's a temporary solution until we are golint compliant in all of our packages. Then I agree, it should be in the Makefile and not the travis file.